lib/Differentiator/DerivativeBuilder.cpp

```diff
@@ -836,6 +863,55 @@ namespace clad {
                     ConstantFolder::synthesizeLiteral(Ty, m_Context, 0));
   }
 
+  StmtDiff ForwardModeVisitor::VisitInitListExpr(const InitListExpr* ILE) {
+    llvm::SmallVector<Expr*, 16> clonedExprs(ILE->getNumInits());
+    llvm::SmallVector<Expr*, 16> derivedExprs(ILE->getNumInits());
+    for (unsigned i = 0; i < ILE->getNumInits(); i++) {
```

Comment:
```suggestion
    for (unsigned i = 0, e = ILE->getNumInits(); i < e; i++) {
```
This way we avoid recomputing the number N times and show the reader that the loop does not mutate the number of the inits of ILE.